### PR TITLE
Add FLYCOLORF7_AIO_HD

### PR DESCRIPTION
**Pull request by haslinghuis**

### CODE REVIEW

File: configs/FLYCOLORF7_AIO_HD/config.h [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+#define BEEPER_PIN PC13
+#define MOTOR1_PIN PC6
+#define MOTOR2_PIN PC7
+#define MOTOR3_PIN PB0
+#define MOTOR4_PIN PB1
+#define MOTOR5_PIN PB6
+#define MOTOR6_PIN PB8
+#define MOTOR7_PIN PB7
+#define MOTOR8_PIN PC9
+#define RX_PPM_PIN PA15
```
Comment (ot0tot) [inline configs/FLYCOLORF7_AIO_HD/config.h:48]: Does this FC really have a PPM input pin?

File: configs/FLYCOLORF7_AIO_HD/config.h [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+#define ADC1_DMA_OPT 1
+
+#define DEFAULT_FEATURES        (FEATURE_OSD | FEATURE_LED_STRIP)
+#define DEFAULT_RX_FEATURE      FEATURE_RX_SERIAL
+
+#define SERIALRX_UART SERIAL_PORT_USART1
+
+#define DEFAULT_PID_PROCESS_DENOM 2
+#define SERIALRX_PROVIDER SBUS
```
Comment (ot0tot) [inline configs/FLYCOLORF7_AIO_HD/config.h:104]: Does this need to be defined?

File: configs/FLYCOLORF7_AIO_HD/config.h [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+#define FC_TARGET_MCU     STM32F7X2
+
+#define BOARD_NAME        FLYCOLORF7_AIO_HD
+#define MANUFACTURER_ID   FLCO
+
+#define USE_ACC
+#define USE_ACC_SPI_MPU6500
+#define USE_BARO
+#define USE_BARO_BMP280
```
Comment (ot0tot) [inline configs/FLYCOLORF7_AIO_HD/config.h:32]: If the baro is on SPI, this should be 
`#define USE_BARO_SPI_BMP280`

File: configs/FLYCOLORF7_AIO_HD/config.h (hunk 1 above)
Comment (haslinghuis) [reply configs/FLYCOLORF7_AIO_HD/config.h:48]: IDK, it even has a timer with dma slapped on it.

File: configs/FLYCOLORF7_AIO_HD/config.h (hunk 2 above)
Comment (haslinghuis) [reply configs/FLYCOLORF7_AIO_HD/config.h:104]: Agree this should be cloud build defined (except for onboard control links using SPI).

File: configs/FLYCOLORF7_AIO_HD/config.h (hunk 3 above)
Comment (haslinghuis) [reply configs/FLYCOLORF7_AIO_HD/config.h:32]: Thanks, good catch.

**Review by nerdCopter — APPROVED** [review]
* approving so that you can merge after resolutions & when you are ready.

**Pull request merged**